config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Bundle.java

```diff
@@ -31,7 +28,24 @@
     private FolderTree folderTree;
     private Map<Dependency, List<Dependency>> dependencyMap;
     private BundleLoadingOperation loadingMode;
-    private Set<BundleDefinedEntities> metadataDependencyBundles;
+    private DependentBundle dependentBundle;
```

Comment:
Why single bundle and list both?

Reply:
dependentBundle is from which this bundle is loaded and list is the set of dependencies this bundle has

Reply:
Should we rename it then?

Reply:
Done